Smarty/templates/Settings/LayoutBlockEntries.tpl

```diff
@@ -14,6 +14,7 @@
 <input type="hidden" name="mode">
 <script type="text/javascript" src="include/js/customview.js"></script>
 <script type="text/javascript" src="include/js/general.js"></script>
+        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
```

Comment:
Please use the jquery library we have in the application:

https://github.com/tsolucio/corebos/tree/master/include/jquery